electron_app/electron.ts

```diff
@@ -416,40 +416,29 @@ function createMainWindow(): void {
   setOpenSolutionsListener();
   setSaveDiagramAsListener();
 
-  const platformIsWindows = process.platform === 'win32';
-  if (platformIsWindows) {
-    browserWindow.webContents.session.on('will-download', async (event, downloadItem) => {
-      const defaultFilename = downloadItem.getFilename();
-
-      const fileTypeIndex = defaultFilename.lastIndexOf('.') + 1;
-      const fileExtension = defaultFilename.substring(fileTypeIndex);
-
-      const fileExtensionIsBPMN = fileExtension === 'bpmn';
-      const fileType = fileExtensionIsBPMN ? 'BPMN (.bpmn)' : `Image (.${fileExtension})`;
-
-      const saveDialogResult = await dialog.showSaveDialog({
-        defaultPath: defaultFilename,
-        filters: [
-          {
-            name: fileType,
-            extensions: [fileExtension],
-          },
-          {
-            name: 'All Files',
-            extensions: ['*'],
-          },
-        ],
-      });
+  browserWindow.webContents.session.on('will-download', (event, downloadItem) => {
+    const defaultFilename = downloadItem.getFilename();
 
-      if (saveDialogResult.canceled) {
-        downloadItem.cancel();
+    const fileTypeIndex = defaultFilename.lastIndexOf('.') + 1;
+    const fileExtension = defaultFilename.substring(fileTypeIndex);
 
-        return;
-      }
+    const fileExtensionIsBPMN = fileExtension === 'bpmn';
```

Comment:
Du könntest hier auch einfach:

```suggestion
    const fileExtensionIsBPMN = defaultFilename.endsWith('.bpmn');
```

machen und die oberen beiden Zeilen wegwerfen.

Reply:
Nein, weil wir die fileExtension ja benötigen